lib/transloadit/assembly.rb

```diff
@@ -66,6 +68,65 @@ def submit!(*ios)
   end
 
   #
+  # alias for create!
+  # keeping this method for backward compatibility
+  #
+  def submit!(*ios)
+    warn "#{caller(1)[0]}: warning: Transloadit::Assembly#submit!"\
+      " is obsolete use Transloadit::Assembly#create! instead"
+
+    self.create!(*ios)
+  end
+
+  #
+  # Returns a list of all assemblies
+  # @param [Hash]        additional GET data to submit with the request
+  #
+  def list(params = {})
+    params = self.to_hash.update(params)
+    Transloadit::Request.new('/assemblies', self.transloadit.secret).get(params)
+  end
+
+  #
+  # Returns a single assembly object specified by the assembly id
+  # @param [String]     id    id of the desired assembly
+  #
+  def get(id)
+    request = Transloadit::Request.new("/assemblies/#{id}", self.transloadit.secret)
+    request.get.extend!(Transloadit::Response::Assembly)
+  end
```

Comment:
This function should actually make two requests for now. One to the general api2.transloadit.com/assemblies/id. And the response of that will contain a property "assembly_url". That endpoint should then be used to get the assembly's status.